src/bson-encode.c

```diff
@@ -648,7 +650,7 @@ static void php_phongo_zval_to_bson_internal(zval* data, php_phongo_field_path*
  * will be used. */
 void php_phongo_zval_to_bson(zval* data, php_phongo_bson_flags_t flags, bson_t* bson, bson_t** bson_out TSRMLS_DC) /* {{{ */
 {
-	php_phongo_field_path* field_path = php_phongo_field_path_alloc();
+	php_phongo_field_path* field_path = php_phongo_field_path_alloc(false);
```

Comment:
Is `php_phongo_field_path* field_path` something that was introduced when you improved BSON exception messages to report the entire field path?

Reply:
Yes.